model/db/heart_disease_bycounty_ddl.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE "State_symbol" (
+	"StateName"	TEXT,
+	"Symbol"	TEXT,
+	PRIMARY KEY("Symbol")
+);
+
+CREATE TABLE "Heart_Disease_Mortality_County" (
```

Comment:
Rename to "State_Counties_Heart_Disease_Mortality"

Reply:
I would change it Heart_Disease_Mortality_State_Counties since the table represents Heart Disease Mortality by state and county